src/plots/piechart.md

```diff
@@ -0,0 +1,64 @@
+# PieChart
+
+The `PieChart` is a _plot operator_ that computes a pie chart for a given vector dataset.
+Moreover, the operator considers all data in the given query rectangle.
+
+There are multiple variants on how to compute the pies.
+In addition, it is possible to compute a donut chart instead of a standard pie chart.
+
+### Parameter
+
+| Parameter    | Type           | Description                                              | Example Value |
+| ------------ | -------------- | -------------------------------------------------------- | ------------- |
+| `type`       | One of `count` | The type of aggregation that is used to create the pies. | `"count"`     |
```

Comment:
There is only one option, right?

Reply:
[9bf291c](https://github.com/geo-engine/docs/pull/37/commits/9bf291c6eb8b25c8ac2a0673ff20e90f5c0b6551)